upstream: unmaintained

| ...build/tmp/sysroots/i686-linux/usr/bin/install: will not overwrite just-created `...build/tmp/work/mipsel-oe-linux/libsigc++-1.2-1.2.5-r2/image/usr/include/sigc++-1.2/sigc++/method_slot.h' with `method_slot.h'

--- libsigc++-1.2.7/sigc++/Makefile.am.orig	2010-09-24 12:03:24.000000000 +0000
+++ libsigc++-1.2.7/sigc++/Makefile.am	2010-09-24 12:03:30.000000000 +0000
@@ -12,7 +12,7 @@
 sigc_built_h	= bind_return.h	\
 		  object_slot.h retype_return.h slot.h bind.h		\
 		  class_slot.h hide.h retype.h signal.h       \
-                  method_slot.h method_slot.h
+                  method_slot.h
 
 built_sources	= $(sigc_built_cc) $(sigc_built_h)
 dist_sources	= $(sigc_fixed_cc) $(sigc_fixed_h)
